workflow/rules/mapping.smk

```diff
@@ -3,25 +3,25 @@ rule map_reads:
         reads=get_merged,
         idx=rules.bwa_index.output
     output:
-        temp("mapped/{sample}.sorted.bam")
+        temp("results/mapped/{sample}.sorted.bam")
     log:
         "logs/bwa_mem/{sample}.log"
     params:
         index=lambda w, input: os.path.splitext(input.idx[0])[0],
         extra=get_read_group,
         sort="samtools",
         sort_order="coordinate"
-    threads: 8
+    threads: 100
```

Comment:
Actually, I tried this once after your advice last time, and it seems this does not saturate at all. If I remember correctly, it was still about 8 cores being used most of the time. Might be because we use samtools to sort, instead of sambamba here.

Reply:
Oh that's weird. Actually this is crucial for me performing WGS analysis. But I replaced this rule with the bwa-samblaster wrapper. I will add a config option to change the cores.

Reply:
You don't need a config option. You can (since some recent release) overwrite the threads of any rule at the command line via e.g. `--threads bwa_mem=100`.